core/iwasm/interpreter/wasm_loader.c

```diff
@@ -7950,10 +7954,12 @@ typedef struct WASMLoaderContext {
     int16 preserved_local_offset;
 
     /* const buffer */
-    uint8 *const_buf;
-    uint16 num_const;
-    uint16 const_cell_num;
-    uint32 const_buf_size;
+    int64 *i64_consts;
```

Comment:
I am wondering how about something like `8_bytes_consts`? Ultimately, we ended up storing a mix of i64 and f64 values, rather than only i64.

Reply:
We treat f64 as i64 by reading its content with `*(int64 *)&f64`, and we sort and binary search the array with i64 elements, I think it is good to name it as `i64_consts`. We can add some comments for it.